### ISSUE TITLE
Update site content

### ISSUE THREAD

**Issue by Dahianna** [18F/18f.gsa.gov#2981]
After a small language update on FY19, we now want to ensure our content still resonates with federal employees and potential staff. 

## To do
- [x] Test content with users
- [x] Update homepage based on feedback
- ~Update what we deliver based on feedback~
- [x] Update how we work based on feedback

**Comment by Dahianna** [comment]
Currently working on this doc: https://docs.google.com/document/d/11IJC3cO79s6rDcu6NH3F0SKSTyjktKvDQ1HkYCtu4B8/edit#

**Comment by lsgitter** [comment]
@margaretmildrew to follow up with @Dahianna on August 12th to see if this is ready to post.

**Comment by Dahianna** [comment]
Doc with changes https://docs.google.com/document/d/18DJxV_yOIDWSb6qcBpn4Z4eUgfCTNkElRZ5I1zFscjs/edit#

**Comment by Dahianna** [comment]
Angela reviewed and we're waiting on some additions from acqstack

**Comment by Dahianna** [comment]
Updates done to home page and what we deliver

**Comment by Dahianna** [comment]
Working on how we work. On hold until Ashley is back

**Comment by Dahianna** [comment]
Working on leadership to update content
- How we work: https://docs.google.com/document/d/16RHE1vOc5pDAmkFlS67Pm56WJeMfygraeOCsmKg-Hes/edit#heading=h.9ewvmwvq5cux
- What we deliver: https://docs.google.com/document/d/1TCPZp6S7Cbxs5qj0O6zgQZBS7bHeH-YppFLiWmSvMb4/edit#

**Comment by apburnes** [comment]
Closing since we are rethinking the What We Deliver content
